support/win_prng_support/src/lib.rs

```diff
@@ -2,6 +2,8 @@
 // Licensed under the MIT License.
 
 //! Support for running in smaller Windows editions such as Win1.
+//! This crate purposefully overrides symbols normally provided by system DLLs.
```

Comment:
nit: you want a newline between the first line and the second line, unless you want the rustdoc to lump these all in on one paragraph.